refactor(VTypeGen): Instruction vtype is generated in IBuffer now

hululu0628 opened the pull request.

hululu0628 committed:
refactor(VTypeGen): Instruction vtype is generated in IBuffer now

eastonman commented:
Wait for tomorrow. Not quite understand VType. Let me discuss with you tomorrow.

hululu0628 committed:
fix(VTypeGen): fix two bugs in VTypeGen and IBuffer

Description:

1. vtype of outputEntries will update even when there no new instructions from ibuffer and decode is no reeady yet.

2. When vtypespec is modified by signals from rob, ibuffer may send new instructions to outputEntries, which makes instructions' vtype wrong.

eastonman commented:
Closed by #6376
